pkg/collector/corechecks/snmp/checkconfig/config.go

```diff
@@ -25,6 +28,12 @@ const defaultOidBatchSize = 5
 const defaultPort = uint16(161)
 const defaultRetries = 3
 const defaultTimeout = 2
+const defaultWorkers = 5
```

Comment:
Given the fact that we should be able monitor thousands of devices, shouldn't we have more workers by default? Would 50 a good fit ?

Reply:
Based on recent discuss that the goal of SNMP Corecheck Discovery is to ease migration from Python AD, I think using the same number of workers as in python makes also sense. (using more workers leads to more mem/cpu usage and might not be expected by users)